src/Symfony/Component/HttpKernel/HttpCache/Store.php

```diff
@@ -408,6 +409,8 @@ private function save($key, $data)
         }
 
         @chmod($path, 0666 & ~umask());
+
+        return true;
```

Comment:
💡 The `save()` method returned either `false` or `null`. Since `null` seems to indicate success, I've changed the method to return `true` instead, which makes checks for the error case easier and allows us to add a `bool` return type later.

Reply:
On second thought, we could very well have `save()` throw directly instead of returning anything. WDYT?

Reply:
Oh, I see, the exception message is different in the first case. Nevermind.